xcode/Subconscious/Shared/Components/Notebook/Notebook.swift

```diff
@@ -497,7 +497,11 @@ struct NotebookModel: ModelProtocol {
         state: NotebookModel,
         environment: AppEnvironment
     ) -> Update<NotebookModel> {
-        return Update(state: state)
+        return update(
+            state: state,
+            action: .refreshLists,
```

Comment:
We don't want to send this until `.ready`, since database will not be ready yet. We already do send this during `.ready`.

Do you recall why this was added?

Reply:
Yes, we need to refresh the notebook when the tab is presented by switching to it and this is the action dispatched when the view is presented. 

We can add an if check to skip the refresh if the DB isn't ready yet?

Reply:
@bfollington that makes sense. Yeah, I think we should add that check. We have an `isDatabaseReady` field on `Notebook` to flag this.

Reply:
Done!